src/main/java/de/hhu/stups/plues/ui/controller/Reports.java

```diff
@@ -185,67 +270,242 @@ private void displayModuleAbstractUnitUnitSemesterConflicts(final Store store,
       }
     });
     moduleAbstractUnitUnitSemesterConflicts.setData(conflicts);
+
+    return conflicts;
```

Comment:
split into a method that computes the data (e.g. in RealReportData) and one step that displays it.